CHANGELOG.md

```diff
@@ -1,8 +1,6 @@
 Changelog
 =========
 
-## TBD
-
```

Comment:
No need to remove this. In fact, its [automatically replaced during the release process with the new version number and date](https://github.com/bugsnag/bugsnag-cocoa/blob/e95c825c64cb359ba87dfec32dfa66a11ce47dc9/Makefile#L67).

Reply:
TIL, thanks.  Added back in.